### ISSUE TITLE
How to render formula in realtime

### ISSUE THREAD

**Issue by zhangmozhe** [jaredly/hexo-admin#121]
Hi, I just try to use this hexo-admin, and enjoy it a lot. Since I want to set up a blog for academic usage, I wonder how to render the formula in the plugin. That is, how to use MathJax during the preview under hexo-admin? 

I believe many academic researchers will face the same problem. Hope to address it soon. Thanks!

**Comment by jaredly** [comment]
There are markdown plugins that transform e.g. `$2 + 2$` into latex, and it wouldn't be too hard to add.

**Comment by PirtleShell** [comment]
Yes, for formula rendering I use MathJax which just surmounts to putting this in your template:
```html
    <script type="text/x-mathjax-config">
      MathJax.Hub.Config({
        tex2jax: {inlineMath: [['$','$'], ['\\(','\\)']]},
        CommonHTML: {scale: 85}
      });
    </script>
    <script type="text/javascript" async
      src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS_CHTML">
```

And then wrapping your tex in `<math>$` and `$</math>`.

These however, will not appear in your live preview. Currently, hexo-admin does not use the same markdown renderer as your frontend (see #22) and there is not currently a way to inject scripts into the editor.

I think if we were to include the frontend markdown renderer, having the ability to extend the editor page with extra code would be a good idea that would allow for inserting your mathjax config info. I'm a little concerned about performance of that and using hexo's renderer on the whole, though..
